test/integration/show_int_test.go

```diff
@@ -24,11 +25,29 @@ func (suite *ShowIntegrationTestSuite) TestShow() {
 
 	suite.PrepareActiveStateYAML(ts)
 
-	cp := ts.Spawn("show")
+	cp := ts.SpawnWithOpts(
+		e2e.WithArgs("activate"),
+		e2e.AppendEnv("ACTIVESTATE_CLI_DISABLE_RUNTIME=false"),
+	)
+	cp.WaitForInput()
+
+	cp = ts.Spawn("show")
 	cp.Expect(`Name`)
 	cp.Expect(`Show`)
 	cp.Expect(`Organization`)
 	cp.Expect(`cli-integration-tests`)
+	cp.Expect(`Namespace`)
+	cp.Expect(`cli-integration-tests/Show`)
+	cp.Expect(`Location`)
+	// MacOS seems to have issues matching this directory
```

Comment:
Could you try a `cp.ExpectLongString()` first? I'm reluctant to omit macOS so quickly, especially after that PythonPath issue with showing tempdir paths.

Reply:
Unfortunately, that didn't work. Good idea though. Not sure what's going on with the expect on MacOS

Reply:
Okay, maybe @Naatan has an idea given he's been working on refactoring termtest.

Reply:
Is it not matching even though it should or is it not matching cause it comes up with a different direction altogether?

For the former; yeah that's an issue with our current termtest implementation. No solution until I can land my rewrite.

For the latter; try using: https://github.com/ActiveState/cli/blob/79f31adbaf3a18072127fb863ec69a45660faceb/internal/fileutils/fileutils_lin_mac.go#L35-L35

Reply:
When I looked at the test output, the directory name looked fine, it was just too long, and the terminal appeared to wrap and indent the output.

Reply:
`ResolveUniquePath` worked! Thanks @Naatan @mitchell-as

Reply:
`ResolveUniquePath` worked! Thanks @Naatan @mitchell-as